tests/test_empty_input.py

```diff
@@ -0,0 +1,36 @@
+import os
+from cStringIO import StringIO
+from tempfile import NamedTemporaryFile
+
+from cwltool.main import main
+from tests.util import needs_docker
+
+@needs_docker
+def test_empty_input():
+    workflow = '''
```

Comment:
Why not use `util.get_data` and keep it external?

Reply:
I followed the style in tool_argparse, I can make them all external